src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -365,30 +363,31 @@ void WebRTCTransportProviderServer::HandleProvideOffer(HandlerContext & ctx, con
 
     WebRTCSessionStruct outSession;
 
-    // New session request: webRTCSessionID is null.
-    if (webRTCSessionID.IsNull())
+    // TODO: this check needs to be removed after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11333
```

Comment:
Again: the check must be here, but after 11333 needs to return INVALID_COMMAND per spec.